test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -9,7 +9,9 @@
 import seedu.addressbook.commands.*;
 import seedu.addressbook.commands.employee.*;
 import seedu.addressbook.commands.member.MemberAddCommand;
-import seedu.addressbook.commands.menu.MenuViewAllCommand;
+import seedu.addressbook.commands.menu.*;
```

Comment:
there's no need to import the specific classes if you have done a ".*" already. This is because the .* means you want to import all the classes in the directory specified already.